client/src/main/java/com/alibaba/nacos/client/logging/AbstractNacosLogging.java

```diff
@@ -36,8 +39,9 @@
     static {
         String loggingPath = System.getProperty(NACOS_LOGGING_PATH_PROPERTY);
         if (StringUtils.isBlank(loggingPath)) {
-            String userHome = System.getProperty("user.home");
-            System.setProperty(NACOS_LOGGING_PATH_PROPERTY, userHome + "/logs/nacos");
+            String cacheDir = CacheDirUtils.getCacheDir();
+            System.setProperty(NACOS_LOGGING_PATH_PROPERTY, cacheDir +  File.separator + "nacos" 
+                    + File.separator + "logs");
```

Comment:
the original path is `user.home/logs/nacos` the new path is `cacheDir/nacos/logs`